packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePlugin.kt

```diff
@@ -179,14 +176,17 @@ class AmplifyDataStorePlugin : FlutterPlugin, MethodCallHandler {
 
         plugin.delete(
                 instance,
-                queryPredicates,
                 Consumer {
                     LOG.debug("Deleted item: " + it.item().toString())
-                    handler.post { flutterResult.success(FlutterSerializedModel(it.item()).toMap()) }
+                    handler.post { flutterResult.success(null) }
                 },
                 Consumer {
                     LOG.debug("Deletion Failed: " + it)
-                    prepareError(flutterResult, it, FlutterDataStoreFailureMessage.AMPLIFY_DATASTORE_DELETE_FAILED.toString())
+                    if (it is DataStoreException && it.localizedMessage == "Wanted to delete one row, but deleted 0 rows.") {
```

Comment:
@jamesonwilliams is there something we can do to make the exception easy to parse instead of doing message parsing which I believe is quite fragile (maybe having an exception code?)

Reply:
@haverchuck & I agree this is not ideal, but it the kludge will work for now.

In the future, we could either:
1. Add an `ItemNotFoundException`, which `extends DataStoreException`. Then you could check `instanceof`, here.
2. Place an `ItemNotFoundException` as the `cause` of the `DataStoreException`. Then you can check `dataStoreException.getCause() instanceof DataStoreException`.